src/ControlSystem/Systems/Translation.hpp

```diff
@@ -78,7 +78,10 @@ struct Translation : tt::ConformsTo<protocols::ControlSystem> {
   static_assert(
       tt::conforms_to_v<measurement, control_system::protocols::Measurement>);
 
-  using control_error = ControlErrors::Translation<NumberOfObjects>;
+  using control_error = ControlErrors::Translation<
+      std::is_same_v<measurement, measurements::BothNSCenters>
+          ? 1
+          : NumberOfObjects>;
```

Comment:
Add a comment above this that even though BNS has two objects, we want to only control the translation via the CoM, not the individual objects, so the control error is only for 1 object.